spec/spec_helper.lua

```diff
@@ -54,3 +47,16 @@ end)
 busted.subscribe({ 'file', 'end' }, function ()
   collectgarbage()
 end)
+
+do
```

Comment:
I think we need to explain what this does, which is memoize ffi.cdef calls.
I also think we need to explain why we need to do this. It's hard to know the reason to do this without any context.

Reply:
Yep.